### PR TITLE
🔖initial commit

### PR DESCRIPTION
**Pull request by dyollb**
initial commit:
- readme
- visual
- license
- code of conduct
- issue template
- pre-commit hook

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug", "triage"]
```
Comment (pcrespov) [inline .github/ISSUE_TEMPLATE/bug_report.yml:4]: I guess you need to add the label ``triage`` in your list https://github.com/ITISFoundation/IXI025/labels

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+assignees:
+  - octocat
```
Comment (pcrespov) [inline .github/ISSUE_TEMPLATE/bug_report.yml]: You also should put a user defined in your repo here e.g. lloyd

File: .github/ISSUE_TEMPLATE/new_issue.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+---
+name: Tracking issue
+about: Use this template for tracking new features.
+title: '[DATE]: [FEATURE NAME]'
+labels: tracking issue, needs triage
```
Comment (pcrespov) [inline .github/ISSUE_TEMPLATE/new_issue.md:5]: Labels need to be in https://github.com/ITISFoundation/IXI025/labels

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 4]
```diff
@@ -0,0 +1,62 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
```
Comment (pcrespov) [inline .github/ISSUE_TEMPLATE/bug_report.yml:3]: NICE! 

TIP: i love adding emoticons in the title. It provides a very nice visual on the commit changes e.g. when you look at the last commit in the code tree. Checkout https://gitmoji.dev/ to get fast an emoticon

I downloaded them all with a script


🔨 Add or update development scripts.
- 🩹 Simple fix for a non-critical issue.
- ⚗️ Perform experiments.
- 👽️ Update code due to external API changes.
- 🚑️ Critical hotfix.
- 💫 Add or update animations and transitions.
- ⬇️ Downgrade dependencies.
- ⬆️ Upgrade dependencies.
- 🎨 Improve structure / format of the code.
- 🍻 Write code drunkenly.
- 🍱 Add or update assets.
- 🔖 Release / Version tags.
- 💥 Introduce breaking changes.
- 🧱 Infrastructure related changes.
- 🐛 Fix a bug.
- 🏗️ Make architectural changes.
- 💡 Add or update comments in source code.
- 👥 Add or update contributor(s).
- 📸 Add or update snapshots.
- 🗃️ Perform database related changes.
- 📈 Add or update analytics or track code.
- 🚸 Improve user experience / usability.
- 🔐 Add or update secrets.
- 🤡 Mock things.
- ⚰️ Remove dead code.
- 🚧 Work in progress.
- 👷 Add or update CI build system.
- 🥚 Add or update an easter egg.
- 🔥 Remove code or files.
- 🌐 Internationalization and localization.
- 🥅 Catch errors.
- 💚 Fix CI Build.
- 🔨 Add or update development scripts.
- ➖ Remove a dependency.
- ➕ Add a dependency.
- 📱 Work on responsive design.
- 🏷️ Add or update types.
- 💄 Add or update the UI and style files.
- 🔒️ Fix security issues.
- 🔊 Add or update logs.
- 🔍️ Improve SEO.
- 📝 Add or update documentation.
- 🧐 Data exploration/inspection.
- 🔇 Remove logs.
- 👔 Add or update business logic
- 📦️ Add or update compiled files or packages.
- 📄 Add or update license.
- 🛂 Work on code related to authorization, roles and permissions.
- ✏️ Fix typos.
- 💩 Write bad code that needs to be improved.
- 📌 Pin dependencies to specific versions.
- ♻️ Refactor code.
- ⏪️ Revert changes.
- 🚀 Deploy stuff.
- 🚨 Fix compiler / linter warnings.
- 🙈 Add or update a .gitignore file.
- 🌱 Add or update seed files.
- ✨ Introduce new features.
- 💬 Add or update text and literals.
- 🩺 Add or update healthcheck.
- 🎉 Begin a project.
- 🧑‍💻 Improve developer experience
- 🧪 Add a failing test.
- 🚩 Add, update, or remove feature flags.
- 🚚 Move or rename resources (e.g.: files, paths, routes).
- 🔀 Merge branches.
- 🗑️ Deprecate code that needs to be cleaned up.
- ♿️ Improve accessibility.
- ✅ Add, update, or pass tests.
- 🔧 Add or update configuration files.
- ⚡️ Improve performance.

File: README.md [hunk 5]
```diff
@@ -1,2 +1,47 @@
...
+[![license: CC BY-SA 3.0](https://licensebuttons.net/l/by-sa/3.0/80x15.png)](https://creativecommons.org/licenses/by-sa/3.0/)
+[![downloads](https://img.shields.io/github/downloads/ITISFoundation/IXI025/total.svg)](<>)
+
+# IXI025 Model - IT'IS Template Head Segmentation Repository
+
+The IXI025 head model is based on Subject 025 from the [IXI dataset](https://brain-development.org/ixi-dataset/). It includes a whole head segmentation, surface-based model, fiducials for 10-10 system construction, as well as the original anatomical (T1w, T2w, PD, MRA) and diffusion weighted images. The head segmentation/model is in the original scanner coordinates, i.e., it is perfectly aligned with the magnetic resonance images.
+
+This dataset is made public to support research in computational brain stimulation modeling, e.g., transcranial electric, magnetic, and acoustic brain stimulation, as well as temporal interference (TI) stimulation. The head model is used for Sim4Life tutorials by IT'IS and its partner organization ZMT Zurich MedTech AG.
+
+![Head segmentation and model](https://raw.githubusercontent.com/ITISFoundation/IXI025/main/visuals/head_model.png)
```
Comment (pcrespov) [inline README.md]: ``Head segmentation and model`` does not work?

**Review by pcrespov — APPROVED** [review]
Very nice!

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (dyollb) [reply .github/ISSUE_TEMPLATE/bug_report.yml:4]: ok, thanks for pointing that out

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 2 above)
Comment (dyollb) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: true, thanks

File: README.md (hunk 5 above)
Comment (dyollb) [reply README.md]: I guess it needs to be merged first.

Comment (dyollb) [reply README.md]: > `Head segmentation and model` does not work?

It does/would! Will fix.

**Pull request merged**